CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.4.7
+* **[Bug Fix]**: Update vscode-webhint to version 1.6.6. Fixes "Unable to start webhint" errors - [PR #951](https://github.com/microsoft/vscode-edge-devtools/pull/951)
+* **[Bug Fix]**: Improve logic for configuring existing launch.json - [PR #945](https://github.com/microsoft/vscode-edge-devtools/pull/

```

Comment:
Missing closing parentheses and pull request number

Reply:
Good catch! Text is fixed and linked VSIX updated (since the VS Code UI uses the changelog from the package).